libs/testing/fixtures/src/lib/application.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Application,
+  ApplicationStatus,
+  ApplicationTypes,
+} from '@island.is/application/core'
+import * as faker from 'faker'
+
+export const createApplication = (
+  overrides: Partial<Application>,
```

Comment:
Can you make it optional for ease of use?

Reply:
Done, good point thanks